app/commands/decidim/half_signup/send_verification.rb

```diff
@@ -13,6 +13,8 @@ def call
         return broadcast(:invalid) unless @form.valid?
 
         if @form.auth_method == "sms"
+          return broadcast(:invalid, "non disponible") if registered_user_with_same_phone_and_country?
```

Comment:
```suggestion
          return broadcast(:invalid, :already_exist) if registered_user_with_same_phone_and_country?
```

Prefer to use symbol rather than french translation